tests/utils.ts

```diff
@@ -105,10 +108,11 @@ export async function openPlot(
   name: PlotNames,
   waitForLoadFinished = true
 ): Promise<void> {
-  await openObject(page, 'plot', name);
+  const panelCount = await page.locator('.iris-chart-panel').count();
+  await openPanel(page, name);
 
   if (waitForLoadFinished) {
-    // Wait until it's done loading
+    await expect(page.locator('.iris-chart-panel')).toHaveCount(panelCount + 1);
     await expect(
       page.locator('.chart-panel-container .loading-spinner')
     ).toHaveCount(0);
```

Comment:
I'd add a check after the loading as well that checks the internal chart wrapper exists. This caused some inconsistency w/ the lazy loading PR.

See https://github.com/deephaven/web-client-ui/pull/1802/files#diff-bccd87d19079cf7b35274281848f7e5a32a070e5101bff9c5b67b58fb82fc16d